components/connect_v2/integration/block_collection/view.coffee

```diff
@@ -24,7 +24,10 @@ module.exports = class BlockCollectionConnectIntegrationView extends Backbone.Vi
 
     this
 
-  remove: ->
+  remove: (e) ->
+    e.stopPropagation()
+    e.preventDefault()
```

Comment:
I think there might not always be an `e` here which means this might fail. Either have another method that calls to it *or* use the existential operator